### ISSUE TITLE
Look for contaminant proteoforms with natural abundance masses (not NeuCode)

### ISSUE THREAD

**Issue by acesnik** [smith-chem-wisc/proteoform-suite#297]

**Comment by acesnik** [comment]
I'm curious whether some components belonging to (unlabeled) contaminants might somehow impact NeuCode quantification or identification. We would need to go look for examples where this is the case before implementing changes.

Secondarily, because contaminants are unlabeled, our theoretical database is incorrect, since we're constructing the contaminant theoreticals as if they were NeuCode labeled. I suggest:
1. We ignore contaminants for NeuCode labeled experiments, or
2. We search for them at natural isotopic abundances

One thing to consider is that decoy "NeuCode" contaminants are getting hits when we use a decoy database.

**Comment by lschaffer2** [comment]
I think it would be a good idea to always look for neucode peaks that could correspond to (unlabeled) contaminants.

**Comment by acesnik** [comment]
One thing we should check is whether there are examples of contaminants that end up in a NeuCode pair. Shortreed stipulates that they're filtered out with the requirement that each pair have a NeuCode ratio of between 1.5 to 6.

**Comment by lschaffer2** [comment]
Yes that's what I meant by NeuCode peaks. Would be interesting to see if that's happening or if the ratio requirement is good enough

**Comment by acesnik** [comment]
We should avoid populating our database with proteins we can't identify to avoid blowing up our FDRs with hits to their decoys. Unlabeled contaminants might be in that category.

**Comment by acesnik** [comment]
@lschaffer2 I like your idea. Maybe instead of trying to filter them, we should initially just mark them as possibly having some influence from contaminants, so that we know where to check.
